qiskit/providers/v2/options.py

```diff
@@ -0,0 +1,36 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Container class for backend options."""
+
+
+class Options:
```

Comment:
If this inherits `SimpleNamespace` or even `dict`, then user can more easily list all available options and use autocomplete.

Reply:
Done in: https://github.com/Qiskit/qiskit-terra/pull/5086/commits/6533f3402a141aaecf3256e82493dee951b00e8d